future/transaction.go

```diff
@@ -1273,6 +1276,36 @@ func MakeApplicationCallTxWithBoxes(
 	return setFee(tx, sp)
 }
 
+// AssignGroupID computes and return list of transactions with Group field set.
+// - txns is a list of transactions to process
+// - account specifies a sender field of transaction to return. Set to empty string to return all of them
+func AssignGroupID(txns []types.Transaction, account string) (result []types.Transaction, err error) {
+	gid, err := crypto.ComputeGroupID(txns)
+	if err != nil {
+		return
+	}
+	var decoded types.Address
+	if account != "" {
+		decoded, err = types.DecodeAddress(account)
+		if err != nil {
+			return
+		}
+	}
+	for _, tx := range txns {
+		if account == "" || bytes.Compare(tx.Sender[:], decoded[:]) == 0 {
+			tx.Group = gid
+			result = append(result, tx)
+		}
+	}
+	return result, nil
+}
+
+// EstimateSize returns the estimated length of the encoded transaction
+func EstimateSize(txn types.Transaction) (uint64, error) {
+	const NumOfAdditionalBytesAfterSigning = 75
```

Comment:
minor: please considering moving this to `future/transaction` level constant, which is the same as `transaction/transaction`

Reply:
done! [4683fb3](https://github.com/algorand/go-algorand-sdk/pull/435/commits/4683fb3dc77e966e9acacc775db0ae9775f9b5d6)